app/controllers/task_controller.rb

```diff
@@ -12,14 +12,14 @@ def show
   end
```

Comment:
To match the rails naming conventions - this controller should be plural, `TasksController`